### ISSUE TITLE
What is expected Kinetics400 dataset directory structure?

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/vision#3273]
Given that the dataset does not come with official downloader scripts and that most roll their own or hack some third-party scripts, it would be much clearer if https://pytorch.org/docs/stable/torchvision/datasets.html#kinetics-400 explained what directory structure is expected by `torchvision.datasets.Kinetics400`

What is the expected dataset size? and the video file extensions?

Thanks!

cc @pmeier

**Comment by vfdev-5** [comment]
@vadimkantorov maybe notes from ref example could help: https://github.com/pytorch/vision/tree/master/references/video_classification#data-preparation

cc @bjuncek

**Comment by vadimkantorov** [comment]
This is not very helpful, since it does not mention explicitly the class-label directory organisation. It is also not clear if root_dir should contain the splits or if the root_dir is the path to a chosen split.

**Comment by vfdev-5** [comment]
Unfortunately, I removed the dataset from my storage where I tried a minimal training of the ref example. 
Maybe, Bruno could share more info about that...

**Comment by vadimkantorov** [comment]
The problem is that the dataset does not have official download or dataset stats scripts or recommended directory structure. From what I found, the root_dir should be the path to the split, the split directory should contain directories for every class labels with videos inside them. But this should better be mentioned explicitly in the docs

**Comment by vfdev-5** [comment]
I agree. I had a similar question in my case... If you'd like to improve the docs, feel free to send a PR.
